.github/workflows/build.yml

```diff
@@ -27,55 +27,125 @@ jobs:
         # requires glibc 2.17, which is older than any of the available
         # environments.
         config:
+          # No Windows debug, Dosen't work on actions (easily)
+          # It's hard to get a python debug lib.
         - os: windows-2019
           preamble: call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
           shell: cmd
+          build-type: RelWithDebInfo
+          run-on-prs: true
+        - os: windows-2019
+          preamble: call "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Enterprise\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
+          shell: cmd
+          build-type: Release
+          run-on-prs: false
         - os: ubuntu-20.04
           shell: bash
-        - os: macos-11
+          build-type: Debug
+          run-on-prs: false
+        - os: ubuntu-20.04
+          shell: bash
+          build-type: RelWithDebInfo
+          run-on-prs: true
+        - os: ubuntu-20.04
+          shell: bash
+          build-type: Release
+          run-on-prs: false
           # MacOS toolchain doesn't search /usr/local by default:
           # https://gitlab.kitware.com/cmake/cmake/-/issues/19120
           # The CMake FindPython module's Python::Python target (used
           # via pybind11::embed in python-bridge-test) transitively adds
           # linker flags to system libs, which fail due to this issue.
+        - os: macos-11
           preamble: export LDFLAGS="-L/usr/local/lib"
           shell: bash
+          build-type: Debug
+          run-on-prs: false
+        - os: macos-11
+          preamble: export LDFLAGS="-L/usr/local/lib"
+          shell: bash
+          build-type: RelWithDebInfo
+          run-on-prs: true
+        - os: macos-11
+          preamble: export LDFLAGS="-L/usr/local/lib"
+          shell: bash
+          build-type: Release
+          run-on-prs: false
+
+        # Dosen't work on actions, It's hard to get a python debug lib.
+        exclude:
+          - config:
```

Comment:
[fix] I don't think this exclusion has any effect, since there is no Debug build of Windows in the list above(?).

Reply:
Aye fair, removed in the restructure.